packages/eas-cli/src/utils/statuspageService.ts

```diff
@@ -0,0 +1,61 @@
+import chalk from 'chalk';
+
+import {
+  StatuspageServiceFragment,
+  StatuspageServiceName,
+  StatuspageServiceStatus,
+} from '../graphql/generated';
+import { StatuspageServiceQuery } from '../graphql/queries/StatuspageServiceQuery';
+import Log, { link } from '../log';
+
+export async function maybeWarnAboutEasOutagesAsync(
+  serviceNames: StatuspageServiceName[]
+): Promise<void> {
+  const services = await getStatuspageServiceAsync(serviceNames);
+
+  for (const service of services) {
+    warnAboutServiceOutage(service);
+  }
+}
+
+const humanReadableServiceName: Record<StatuspageServiceName, string> = {
+  [StatuspageServiceName.EasBuild]: 'EAS Build',
+  [StatuspageServiceName.EasSubmit]: 'EAS Submit',
+  [StatuspageServiceName.EasUpdate]: 'EAS Update',
+};
+
+function warnAboutServiceOutage(service: StatuspageServiceFragment): void {
+  if (service.status === StatuspageServiceStatus.Operational) {
+    return;
+  }
+
+  const outageType = service.status === StatuspageServiceStatus.MajorOutage ? 'major' : 'partial';
+
+  Log.warn(
+    `Service status warning: ${chalk.bold(
+      humanReadableServiceName[service.name]
+    )} is experiencing a ${outageType} outage. The service may temporarily not function as expected.`
```

Comment:
i'd put the "is experiencing a ${outageType} outage" part inside of the bold.

also, i think we can just remove the other text now that we let people know what kind of outage is occurring:

```suggestion
    )} is experiencing a ${outageType} outage.`
```